pallets/delegation/src/lib.rs

```diff
@@ -130,6 +130,7 @@ decl_error! {
 		ParentNotFound,
 		UnauthorizedRevocation,
 		UnauthorizedDelegation,
+		ExeededBounds,
```

Comment:
Typo here. I'd also suggest to get a more descriptive name like
```suggestion
		ExceededRevocationBounds,
```

Reply:
fixed in 4966e4b